### PR TITLE
Bring CI up-to-date.

### PR DESCRIPTION
**Pull request by jsha**
We no longer need to check out the acme-v2 certbot branch. Also, copy in
the latest chisel2 from Boulder.

### CODE REVIEW

**Comment by cpu** [comment]
Aw man! I totally forgot I approved this PR and didn't merge it. I was _just_ about to start doing this same work on my own branch :blush: Sorry @jsha !

File: test/chisel2.py [hunk 1]
```diff
@@ -36,19 +36,24 @@
 logger = logging.getLogger()
 logger.setLevel(int(os.getenv('LOGLEVEL', 0)))
 
-DIRECTORY = os.getenv('DIRECTORY', 'https://localhost:14000/dir')
+DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:14000/dir')
```
Comment (cpu) [inline test/chisel2.py]: I think this needs to be `https://` not `http://`

**Comment by jsha** [comment]
Done!

**Comment by cpu** [comment]
Thanks @jsha !~

**Pull request merged**